README.Rmd

```diff
@@ -57,130 +55,169 @@ library(sfheaders)
 library(blockCV)
 library(yardstick)
 library(snakecase)
+library(patchwork)
 ```
 
```

Comment:
It could be nice to drop a high-level outline of how the process works here, even if it's just dropping the headers into bullet points.